server.js

```diff
@@ -25,6 +25,9 @@ var stats = {
     tunnels: 0
 };
 
+// holds the CLI options
+var options;
```

Comment:
Because it needs to be accessed in two different function scopes and the closest common parent was file scope.